<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimKartlar</class>
 <widget class="QDialog" name="SimKartlar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #DFE8CC ;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="labelSimKartlar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sim Kartlar</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="numara">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Numara</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="numara_">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="kartSeriNo">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Kart Seri No</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="kartSeriNo_">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="kota">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Kota</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="kota_">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="taahhut">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Taahhüt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="taahhut_">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="musteri">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Müşteri</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="musteri_">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
border:2px solid grey;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="tarih">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Tarih</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tarih_">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSil">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButtonSil{
background-color: transparent;
border-image: url(:/img/minusIcon.png);
background: none;
border: none;
background-repeat: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEkle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButtonEkle{
background-color: transparent;
border-image: url(:/img/plusIcon.png);
background: none;
border: none;
background-repeat: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
